libs/ledger-live-common/src/api/socket.ts

```diff
@@ -33,16 +33,11 @@ export const createDeviceSocket = (
   }
 ): Observable<SocketEvent> =>
   new Observable((o) => {
-    let deviceError: Error | null = null; // the socket was interrupted by device problem
-
+    let deviceError: Error | null = null; // error originating from device (connection/response/rejection...)
     let unsubscribed = false; // subscriber wants to stops everything
-
     let correctlyFinished = false; // the socket logic reach a normal termination
-
-    let inBulk = false; // bulk is a mode where we have many apdu to run on device and no longer need the connection
-
-    let timeoutForAllowManager: NodeJS.Timeout | null | number = null; // track if there is an ongoing allow manager step
-
+    let inBulkMode = false; // we have an array of apdus to exchange, without the need of more WS messages.
+    let allowSecureChannelTimeout: NodeJS.Timeout | null | number = null; // track if there is an ongoing allow manager step
```

Comment:
i'm picky 😈 : `track if there is an ongoing secure channel communication step` or equivalent for the comment ;)

Reply:
it's for the user confirmation but indeed the comment should've been updated with the rename